### DISCUSSION TITLE
Unable to reverify academic affiliation to apply for the Campus Expert Program

### DISCUSSION THREAD

**Discussion by VirtualHorror** [community/community#63364] [GitHub Education] [answered]
### Hi everyone,

I recently moved education institutions and was looking forward to apply for Github Campus Expert Program.

To make sure that my academic affiliation is updated in Github's Education record before I applied for the program, I tried reverify my academic affiliation but it does not allow me to proceed stating the error: Applicants of my institution are required to use the school-issued email address.
I reached out to a few professors and understand that the email listed in the error is not provided to students and suggested that I submit admission documents instead.

Unfortunately, no option to upload academic affiliation documents is displayed on the reverification page due to the error it is displaying.
I opened the ticket #2267516, 2 weeks ago knowing I would have this issue when applications for Campus Expert program are open but have not heard back yet.

Would appreciate if someone from the Community or Education team can bring the ticket to the attention of appropriate teams and expedite the request.
cc: @juanpflores @samus-aran @lasr21 @queenofcorgis

**Comment by lasr21** [comment]
Hi @VirtualHorror I see that you have the pack approved :)

**Reply by VirtualHorror** [reply]
Hello @lasr21 I already have the pack approved but the issue is that I have moved educational institutions and applying for the Campus Expert Program without reverifying my academic credentials would create confusion for the Education team when reviewing my application because they would still think I am affiliated to my previous institution

When I try to reverify my academic credentials it requires a school-issued email address but like I mentioned earlier, those are not provided to students. I would want to proceed with uploading academic affiliation documents rather than the school-issued email address but there is no option to upload them.
Would be grateful if you could review ticket #2267516 where I have explained the situation in detail.

**Comment by lasr21** [comment, marked as answer]
We understand the challenges you're facing with verifying your academic credentials due to the email address issue. I want to reassure you that the name of your school will not be a factor in the initial round of application reviews for the GitHub Campus Expert program. Our review process is blind, focusing on the skills and qualities you bring to the table rather than your academic affiliation.

If you are selected for the program, we will request further details about your academic program and school at a later stage. So, please don't worry about the email verification issue at this moment.

**Reply by VirtualHorror** [reply]
Really appreciate your help throughout the application process, thank you Luis.
